src/components/title-show.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import IconButton from '@folio/stripes-components/lib/IconButton';
+import PaneMenu from '@folio/stripes-components/lib/PaneMenu';
+
+import DetailsView from './details-view';
+import KeyValueLabel from './key-value-label';
+import ScrollView from './scroll-view';
+import PackageListItem from './package-list-item';
+import IdentifiersList from './identifiers-list';
+import ContributorsList from './contributors-list';
+
+export default function TitleShow({ model }, { router, queryParams }) {
+  let historyState = router.history.location.state;
+
+  return (
+    <DetailsView
+      type="title"
+      model={model}
+      showPaneHeader={!queryParams.searchType}
+      paneHeaderFirstMenu={historyState && historyState.eholdings && (
+        <PaneMenu>
+          <div data-test-eholdings-title-show-back-button>
+            <IconButton icon="left-arrow" onClick={() => router.history.goBack()} />
+          </div>
+        </PaneMenu>
+      )}
+      bodyContent={(
+        <div>
```

Comment:
not a blocker, but i wonder if it's possible to extract the JSX we pass to `bodyContent` to somewhere else - maybe a separate file with an export or a variable somewhere else in the component.  inlining it can make the `DetailsView` component a bit unwieldy with respect to readability imo, especially for larger detail views.

Reply:
I thought of this too. I think we can keep them in the same file, but just in a separate function. This way we can read the body function separate, and this prop would look like:
```
bodyContent={titleBodyContent(model)}
```